webpack/global_test_setup.js

```diff
@@ -16,7 +16,7 @@ global.console.error = (error, stack) => {
   //    'The prop `ouiaId` is marked as required in `Modal`, but its value is `undefined`',
   //    'created by WizardHeader', 'created by Navigation',
   //    'created by ActionsColumn', 'created by InactiveText', 'created by Select',
-  //    'created by Context.Consumer'];
+  //    'created by Context.Consumer', 'created by DualListSelector'];
```

Comment:
What is `DualListSelector`

Reply:
It's this patternfly component: https://www.patternfly.org/v4/components/dual-list-selector/

We use it on ACS creation wizard to select products and smart proxies. The component internally renders some child components which are ouia-id enabled but we can't set those in our code directly. Hence ignoring those for now.